src/library_openal.js

```diff
@@ -1917,6 +1920,11 @@ var LibraryOpenAL = {
     c.capturedFrameCount = 0;
     c.isCapturing = false;
 
+    // Disabling the microphone of the browser.
+    AL.mediaStreamTrack.getTracks().forEach(function(track) {
+      track.stop();
+    });
+
```

Comment:
Two issues:
1) Should clear out `AL.mediaStreamTrack = null;` here so that it will be garbage collected on capture device close?
2) `mediaStreamTrack` should be per-capture device specific, and not global per-process? I.e. store the variable as a member of `newCapture` and not global on `AL`. That will also resolve the first issue, since `alcCaptureCloseDevice` does `delete AL.captures[deviceId];`.

Reply:
I've got it.
Now I move `mediaStreamTrack` to `AL.captures`.